Resources/Locale/en-US/_Goobstation/changeling/store/changeling-catalog.ftl

```diff
@@ -1,140 +1,125 @@
-
-# combat
+# Combat
 
 evolutionmenu-combat-armblade-name = Arm Blade
 evolutionmenu-combat-armblade-desc =
-    Reform one of your arms into a grotesque blade, composed of bone and flesh, able to pry open airlocks and cut through your foes like butter.
-    Costs 15 chemicals.
+    We reform one of our arms into a grotesque blade, capable of prying open airlocks and maiming our enemies.
+    Costs a small amount of chemicals.
 
 evolutionmenu-combat-boneshard-name = Bone Shard
 evolutionmenu-combat-boneshard-desc =
-    Break off shards of your bone and shape them into a throwing star which embeds into your foes. But a one timer opportinuty.
-    Costs 15 chemicals.
+    We break off shards of sharp bone to use as projectile weapons.
+    Costs a small amount of chemicals.
 
 evolutionmenu-combat-armor-name = Chitinous Armor
 evolutionmenu-combat-armor-desc =
-    Inflate your body into an all-consuming chitinous mass of armor.
-    Provides extensive protection against physical damage, but less against other types.
-    It massively slows your movement, and maintaining its shape slows chemical generation.
-    WARNING: Requires you to absorb at least 2 organics to use the ability.
-    Costs 25 chemicals.
+    We turn our skin into tough chitin to protect us from damage. Upkeep of the armor requires a low expenditure of chemicals.
+    WARNING: We must absorb at least 2 organics to use this ability.
+    Costs a small amount of chemicals.
```

Comment:
this entry in particular is a little redundant, it mentions chemical upkeep twice. i would either get rid of "costs a small amount of chemicals" OR change it so its something like "Drains a small but consistent amount of chemicals", removing the other line about the upkeep

Reply:
it actually doesnt even work this way so im just removing the text about chemical drain